openmrs/apps/reports/GEORGETOWN_LABOUR_AND_DELIVERY_REPORT/sql/georgetownLabourAndDeliveryReport.sql

```diff
@@ -0,0 +1,26 @@
+SELECT
+    CAST(@a:=@a+1 AS CHAR) as "Serial Number",
+    getFacilityName() as "Facility Name",
+    getPatientIdentifier(r.person_a) as "Patient ID",
+    getPatientAgeInYearsAtDate(r.person_a, NOW()) as "Age",
+    getObsDatetimeValue(r.person_b, "7b88e743-c39f-44cb-8181-9c4eadcf2e12") as "Date of delivery",
```

Comment:
@ASomaroo for this specific column it's better to use the function **getObsDatetimeValueInSection** because the data we are trying to capture are in 2 forms so to be specific rather use that function